zsh-nvm.plugin.zsh

```diff
@@ -71,6 +71,12 @@ _zsh_nvm_load() {
   }
 }
 
+_zsh_nvm_completion() {
+
+  # Add provided nvm completion
+  [[ -r $NVM_DIR/bash_completion ]] && \. $NVM_DIR/bash_completion
```

Comment:
Can you use `source` here instead of `.` here for readability?

Out of interest, why do you prefix `.` with `\`?

Reply:
Fine for me, coming soon.
I was wondering it too for the `\`but as it was in the nvm documentation, I prefer to not to take any risks ;)